### ISSUE TITLE
simple Resource object documentation incorrect and does not work

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3147]
|<img alt="eric_jahn's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| eric_jahn reported|
|-|-|
|Trac ID|trac#3147|
|Type|defect|
|Created|2008-04-01 23:33:24Z|
|Branch|https://github.com/twisted/twisted/tree/webhowto-fix-3147|

The simple Resource object example in the documentation at http://twistedmatrix.com/projects/web/documentation/howto/using-twistedweb.html#auto0
is incorrect.   Some indentation is wrong on the get_request method and some other minor things which cause it not to work.  Here is the correct code for that: http://pastebin.org/26793  (also attached as a file)

Attachments:

* [SimpleResourceObject](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d7e/d7e94e1f1485e03015d366e46ad19ce7e0857d3a/13d806de86b96b95d29392b2b50de7766939b7e4) (333 bytes) - added by eric_jahn on 2008-04-01 23:35:01Z - changed code snippet for simple Resource object example

<details><summary>Searchable metadata</summary>

```
trac-id__3147 3147
type__defect defect
reporter__eric_jahn eric_jahn
priority__normal normal
milestone__ 
branch__branches_webhowto_fix_3147 branches/webhowto-fix-3147
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1207092804000000 1207092804000000
changetime__1207658081000000 1207658081000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [23168]) Branching to 'webhowto-fix-3147'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Thanks!  I put the attached file into a branch, made some changes to make it valid lore markup, and it is now ready for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

Please merge

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [23238]) Merge webhowto-fix-3147

Author: eric_jahn, exarkun
Reviewer: therve
Fixes [#3147](https://github.com/twisted/twisted/issues/3147)

Change the sample code in one of the twisted.web howtos so that creates
a resource which can actually respond to requests and demonstrate some
bit of functionality.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
